ts/features/bonus/bpd/saga/orchestration/onboarding/startOnboarding.ts

```diff
@@ -106,7 +106,7 @@ export function* bpdStartOnboardingWorker() {
  */
 export function* handleBpdStartOnboardingSaga(): SagaIterator {
   const { cancelAction } = yield race({
-    onboarding: call(bpdStartOnboardingWorker),
+    activation: call(bpdStartOnboardingWorker),
```

Comment:
is this rename needed?

Reply:
I found it more relevant calling it `activation`, since we are activating a card not properly requesting an onboarding, what do you think?

Reply:
Sorry i noticed now it was a rename on BPD saga, no it is not...IDE just exceeded in refactoring 😄

Reply:
Fixed on 
ce99b47